npm install fails

When I run:


    $ npm i --save ready-primes

I get
```
> ready-primes@1.0.2 postinstall /Volumes/Dev/primestore/node_modules/ready-primes
> npm run generate


> ready-primes@1.0.2 generate /Volumes/Dev/primestore/node_modules/ready-primes
> node bin\generate.js -w

module.js:472
    throw err;
    ^

Error: Cannot find module '/Volumes/Dev/primestore/node_modules/ready-primes/bingenerate.js'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.runMain (module.js:605:10)
    at run (bootstrap_node.js:427:7)
    at startup (bootstrap_node.js:151:9)
    at bootstrap_node.js:542:3
```

Could it be because of the backslash in `bin\generate.js`?

I am using
- node v7.10.0
- npm v4.6.1
- macOS 10.12.4

I'll try to fix it and send a pull request.

--- a/package.json
+++ b/package.json
@@ -9,7 +9,7 @@
   "scripts": {
     "test": "mocha",
     "postinstall": "npm run generate",
-    "generate": "node bin\\generate.js -w"
+    "generate": "node bin/generate.js -w"
   },
   "repository": {
     "type": "git",

